os/drivers/testcase/testcase_drv.c

```diff
@@ -22,6 +22,7 @@
 #include <tinyara/config.h>
 #include <errno.h>
 #include <debug.h>
+#include <../pm/pm_test.h>
```

Comment:
Please don't use relative path.
There are two methods
1. use CFLAG
2. use include folder

No favorite among them to me.